### ISSUE TITLE
Some validation items not working

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#44]
Hi,

Using the latest homebrew build for Mac, some Validation menu items are not working. Rotamers and B-factor are working, but the Ramachandran plot, Validation outliers and Ramachandran outliers do not display. Looking in the console, there are some errors when I click on these items:

```
ERROR:: glade file /usr/local/Cellar/coot/HEAD-7fb49cd/share/coot/dynarama.glade not found
WARNING:: could not initialise ramachandran

Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/coot/extensions.py", line 121, in <lambda>
    lambda func: validation_outliers_func())
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/coot/extensions.py", line 118, in validation_outliers_func
    find_baddies.validation_outliers_dialog(aa_imol, imol_map)
NameError: name 'find_baddies' is not defined
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/coot/extensions.py", line 124, in <lambda>
    lambda func: rama_outlier_gui())
NameError: name 'rama_outlier_gui' is not defined

**Comment by pemsley** [comment]
@olibclarke Several problems there.  I think that I've found the `dynarama.glade` problem at least.

**Comment by Species10C** [comment]
-I get the same errors as @olibclarke 

But -Overlaps, Peptides, Cbeta, Rama & Rota Outlier panel - this one opens 

---

- Under Density fit analysis popup I now see a Resize setting. With it I can now resize the bars and click-to-go to the residue
- Under Rotamer, no resize button, and not all residues I can click (only red ones).

Seems like it is an issue with the size of the bar?

**Comment by pemsley** [comment]
@Species10C Maybe all bars should be at least 10 pixels high.

**Comment by pemsley** [comment]
The "Overlaps, Peptides, Cbeta, Rama & Rota Outlier panel" dialog is now fixed (1.0.03).
